### PR TITLE
[WIP] rawkv: add mget/mput.

### PR DESCRIPTION
**Pull request by disksing**
cc @zhangjinpeng1987 @AndreMouche @siddontang

### CODE REVIEW

File: src/server/grpc_service.rs [hunk 1]
```diff
@@ -621,6 +628,80 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
         ctx.spawn(future);
     }
 
+    fn raw_m_get(&self,
```
Comment (zhangjinpeng1987) [inline src/server/grpc_service.rs]: s/raw_m_get/raw_mget

File: src/server/grpc_service.rs [hunk 2]
```diff
@@ -621,6 +628,80 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
...
+            .map(|_| timer.observe_duration())
+            .map_err(move |e| {
+                debug!("{} failed: {:?}", label, e);
+                GRPC_MSG_FAIL_COUNTER.with_label_values(&[label]).inc();
+            });
+
+        ctx.spawn(future);
+    }
+
+    fn raw_m_put(&self,
```
Comment (zhangjinpeng1987) [inline src/server/grpc_service.rs]: ditto

File: src/server/grpc_service.rs (hunk 1 above)
Comment (disksing) [inline src/server/grpc_service.rs]: It's generated code. If we want `raw_mget` in rust, we would have to change kvproto/golang from `RawMGet` to `RawMget`. What's your opinion?

**Comment by siddontang** [comment]
I think we don't need it at present, we can close it and reopen if we need it later. 

@disksing

**Pull request closed without merging**